### ISSUE TITLE
Config advice

### ISSUE THREAD

**Issue by RighteousIdealizedDung** [wa0x6e/ResqueBoard#33]
I have php-resque-ex installed and processing jobs. I installed ResqueBoard to monitor those workers but I am not getting any data. From my reading of the install guide I understood that I needed to amend my worker init code to use the cube handler; leaving ResqueBoard options at the defaults they would get the data from cube. My worker init script is below. ResqueBoard is running and connected to redis, mongo & cube. I ran it in debug mode, and the output is below also. The workers are running and processing jobs. Could you please advise what steps I should take next?

Mongo
Query   Took (ms)
CONNECTION to localhost:27017
From /var/www/byta-repository/Symfony/web/ResqueBoard/src/ResqueBoard/Lib/Service/Mongo.php @ 57
0.19
SELECTDB cube_development
From /var/www/byta-repository/Symfony/web/ResqueBoard/src/ResqueBoard/Lib/ResqueStat.php @ 720
0.01
SELECTDB cube_development
From /var/www/byta-repository/Symfony/web/ResqueBoard/src/ResqueBoard/Lib/ResqueStat.php @ 754
0.02
SELECTCOLLECTION cube_development jobs_repartition_stats
From /var/www/byta-repository/Symfony/web/ResqueBoard/src/ResqueBoard/Lib/ResqueStat.php @ 764
0.02
SELECTCOLLECTION cube_development got_events
From /var/www/byta-repository/Symfony/web/ResqueBoard/src/ResqueBoard/Lib/ResqueStat.php @ 766
0.01
Redis
Query   Took (ms)
CONNECTION to 127.0.0.1:6379
From /var/www/byta-repository/Symfony/web/ResqueBoard/src/ResqueBoard/Lib/Service/Redis.php @ 71
0.04
# !/bin/sh

QUEUE=job-dispatcher LOGHANDLER='cube' LOGHANDLERTARGET='udp://127.0.0.1:1180' APP_INCLUDE=/var/www/byta-repository/Symfony/vendor/autoload.php VVERBOSE=1 php /var/www/byta-repository/Symfony/vendor/kamisama/php-resque-ex/bin/resque >> /var/log/byta_resque_dispatcher.log 2>&1 < /dev/null &
QUEUE=job-worker LOGHANDLER='cube' LOGHANDLERTARGET='udp://127.0.0.1:1180' APP_INCLUDE=/var/www/byta-repository/Symfony/vendor/autoload.php VVERBOSE=1 php /var/www/byta-repository/Symfony/vendor/kamisama/php-resque-ex/bin/resque >> /var/log/byta_resque_worker.log 2>&1 < /dev/null &

**Comment by wa0x6e** [comment]
Is your data really sent to Cube safely ?

The best way to confirm if the data are really delivered to Cube safely is to check that there is some data in the `cube_developement` database.

You can either do it by browsing mongo via the cli, or install something like rockmongo.

**Comment by RighteousIdealizedDung** [comment]
Thanks for the response. I am getting data to Cube. I can now see data in ResqueBoard dashboard. However, it seems unreliable. It identifies all the workers but on each refresh I get very different figures for jobs processed. I don't think I will look into this further as my Resque queues are running fine, the ResqueBoard would have been a "nice-to-have" add-on to that but it seems to be taking too much time to configure and debug, so I shall close this issue.

**Comment by wa0x6e** [comment]
There is a old bug in the dashboard not properly displaying datas when the dashboard and resque are not in the same timezone.

Also, if you're refreshing the page, there is a time lag of 5 seconds between the computed datas and realtime data, because cube is built like that.

**Comment by RighteousIdealizedDung** [comment]
Thanks again for your response, and for your development. I may revisit this at some point, bearing in mind your comments, but for this project I'm out of time. There are other problems I am seeing which I didn't mention, eg, a superfluous slash in the htaccess file, 
RewriteRule    (.*) src/ResqueBoard//webroot/$1 [L]

A couple of 404s noted in the console, to /ResqueBoard/src/ResqueBoard/webroot/js/libs/cal-heatmap.source-map.js and /ResqueBoard/src/ResqueBoard/webroot/js/libs/jquery.min.map

None of the links from the dashboard working for me, all lead to pages with no data.

So in short I just think it would take me too much time to get this working for me reliably. But again, many thanks.
